### PR TITLE
GSoC 2017 Post: Native Julia ODE solvers

### PR DESCRIPTION
**Pull request by YingboMa**
cc: @ChrisRackauckas @jiahao @cortner

### CODE REVIEW

File: blog/_posts/2017-09-04-gsoc-ode.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title:  "GSoC 2017: Native Julia ODE solvers"
```
Comment (ChrisRackauckas) [inline blog/_posts/2017-09-04-gsoc-ode.md]: Far too generic of a title. The title should clearly indicate that this is about second order ODEs and BVPs

File: blog/_posts/2017-09-04-gsoc-ode.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title:  "GSoC 2017: Native Julia ODE solvers"
+title:  "GSoC 2017: Native Julia second order ODEs and BVPs solvers"
```
Comment (ChrisRackauckas) [inline blog/_posts/2017-09-04-gsoc-ode.md]: ODE and BVP

**Comment by ViralBShah** [comment]
Folks - what is holding this one? Good to go? If so, just needs a date update.

**Comment by ChrisRackauckas** [comment]
Looks like we need to update the dates on the images as well. Other than that, it looks good to go. I'll edit it right now.

**Pull request merged**